test/common.h

```diff
@@ -232,6 +234,10 @@ static cache_t *create_test_cache(const char *alg_name,
     cache = S3FIFO_init(cc_params, "move-to-main-threshold=2");
   } else if (strcasecmp(alg_name, "Sieve") == 0) {
     cache = Sieve_init(cc_params, NULL);
+  } else if (strcasecmp(alg_name, "Mithril") == 0) {
```

Comment:
Does this need to be updated to use `-p` parameter?

Reply:
I intended to add a Mithril test to this place, and add a `testPrefetchAlgo.c` file to test all prefetch algorithms, but due to time reasons, I did not complete the test work.

Reply:
Sounds good! Will you have time to finish (no rush)? 
BTW, thank you for fixing the leak in admission module :)

Reply:
Yes, I think I can try it! But need a little time.

Reply:
Great! There is no rush! Let me know if you need any help from me.